contracts/src/cosmos/precompile/distribution.sol

```diff
@@ -1,37 +1,68 @@
-// SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
files in this package should be MIT, not BUSL

Reply:
done